front/pages/w/[wId]/builder/assistants/[aId]/index.tsx

```diff
@@ -108,13 +109,30 @@ export async function buildInitialState({
     }
   }
 
-  const databaseQueryConfiguration: AssistantBuilderInitialState["databaseQueryConfiguration"] =
-    null;
+  let tablesQueryConfiguration: AssistantBuilderInitialState["tablesQueryConfiguration"] =
+    {};
```

Comment:
> Why depart from null? Would love to see the approach aligned between actions?

It is aligned with the `dataSources` action. These 2 are similar, we need an object because you can add/remove specific sources (and you can have several). 
Makes the builder code simpler to just initialize right away with the correct thing instead of allowing null and having to check it everywhere.

Reply:
Gotcha :+1: